### PR TITLE
Fix failures with ansible-test-sanity-docker-.*

### PR DESCRIPTION
**Pull request by NilashishC**

### CODE REVIEW

File: plugins/module_utils/network/nxos/config/acls/acls.py [hunk 1]
```diff
@@ -656,7 +656,7 @@ def set_commands(self, want, have):
                                         # remove existing rule to update it
                             elif w_acl.get("aces"):
                                 # 'have' has ACL defined without any ACE
-                                ace_list = [item for item in w_acl["aces"]]
+                                ace_list = list(w_acl["aces"])
```
Comment (ashwini-mhatre) [inline plugins/module_utils/network/nxos/config/acls/acls.py:659]: Are this changes related to fixing sanity?

Comment (NilashishC) [reply plugins/module_utils/network/nxos/config/acls/acls.py:659]: Yeah, pylint was failing with `unnecessary use of comprehension`.

**Comment by ashwini-mhatre** [comment]
LGTM
